tempodb/encoding/vparquet/block_traceql_test.go

```diff
@@ -201,6 +201,11 @@ func TestBackendBlockSearchTraceQL(t *testing.T) {
 	}
 
 	for _, req := range searchesThatMatch {
+		if req.SecondPass == nil {
+			req.SecondPass = func(s *traceql.Spanset) ([]*traceql.Spanset, error) { return []*traceql.Spanset{s}, nil }
```

Comment:
This looks like the default behavior of `MustExtractFetchSpansRequestWithMetadata` already. Is this still needed?

Reply:
Some of the test cases construct conditions directly and don't use `MustExtractFetchSpansRequestWithMetadata`. Honestly I think i'm going to revert the changes to `MustExtractFetchSpansRequestWithMetadata` and just leave this.

Reply:
Ah that's right.  For OpNone which is outside of the language spec for now, but will be covered by `select()` and `by()` very soon.